src/main/Main.java

```diff
@@ -0,0 +1,56 @@
+package main;
+
+import java.util.Scanner;
+
+public class Main {
+    private static final Scanner sc = new Scanner(System.in);
+
+    public static void main(String[] args) {
+        int op;
+        do {
+menu();
+op = sc.nextInt();
+sc.nextLine();
+gerenciaOpcao(op);
+        } while (op != 6);
+
+        sc.close();
+    }
+
+    public static void menu() {
+        System.out.println("\tBem-vindo ao nosso banco!");
+        System.out.println("\tDigite (1) para cadastrar cliente;");
+        System.out.println("\tDigite (2) para sacar;");
+        System.out.println("\tDigite (3) para realisar depósito;");
+        System.out.println("\tDigite (4) para realisar transferencia;");
```

Comment:
Correção de ortografia (trocar a letra "s" pela letra "z" na palavra "realisar" 
```suggestion
        System.out.println("\tDigite (3) para realizar depósito;");
        System.out.println("\tDigite (4) para realizar transferencia;");
```